week-1/exercises/random.js

```diff
@@ -3,6 +3,8 @@ const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
+console.log(num);
+
 // In this exercise, you will need to work out what num represents?
 // Try breaking down the expression and using documentation to explain what it means
```

Comment:
Here, try to explain what the expression on line 4 is doing. Run the program a few times - do you get the same value or different values? Also, try breaking down each part of the expression to help understand